### PR TITLE
PP-7965 Kebabify file names

### PR DESCRIPTION
**Pull request by stephencdaly**
Rename files to match conventions here https://pay-team-manual.cloudapps.digital/manual/coding-standards/node-js-style-guide.html#file-naming-conventions

Did the rename and updating usages in separate commits in an attempt to preserve the file history in git. When both are done at once, it doesn't recognise all the changes as renames.

### CODE REVIEW

File: app/metrics/graphite-reporter.js [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-const graphiteReporter = new metrics.GraphiteReporter(registry, "frontend", process.env.METRICS_HOST, process.env.METRICS_PORT || 8094)
+const registry = new metrics.Report()
+const graphiteReporter = new metrics.GraphiteReporter(registry, 'frontend', process.env.METRICS_HOST, process.env.METRICS_PORT || 8094)
 graphiteReporter.start(1000)
 
-module.exports.getCounter = function getCounter(metricsPrefix) {
+module.exports.getCounter = function getCounter (metricsPrefix) {
   if (!registry.getMetric(metricsPrefix)) {
-    const counter = new metrics.Counter
+    const counter = new metrics.Counter()
```
Comment (danworth) [inline app/metrics/graphite-reporter.js:11]: Didn't know it would even work without the `()`

File: app/utils/email-tools.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ const validEmail = email => {
   const match = reValidEmail.exec(email)
   return {
     'local-part': match !== null ? match[1] : false,
```
Comment (danworth) [inline app/utils/email-tools.js:27]: curious why `local-part` isn't updated to remove the `'` as per `domain` below, is it because of the `-`?

**Review by danworth — APPROVED** [review]
Nice tidying and linting 🏅  There is a lot of change here albeit pretty simple. I'm confident its ok because the CI passing, the scripts wouldn't load if they couldn't find the newly named files and I assume the linting changes were suggested by `standard`

File: app/utils/email-tools.js (hunk 2 above)
Comment (stephencdaly) [reply app/utils/email-tools.js:27]: yeh it will be because of that

**Pull request merged**